src/patternfly/components/NavSystem/styles.scss

```diff
@@ -0,0 +1,481 @@
+  // font
+  --pf-c-nav__list-tertiary-link--FontWeight:                           var(--pf-global--link--normal);
+  --pf-c-nav__list-tertiary-link--active--FontWeight:                   var(--pf-global--FontWeight--bold);
+  // color
+  --pf-c-nav__list-tertiary-link--Color:                                var(--pf-global--Color--dark-100);
+  --pf-c-nav__list-tertiary-link--active--Color:                        var(--pf-global--link--Color);
+  --pf-c-nav__list-tertiary-link--disabled--Color:                      var(--pf-global--disabled-color--100);
+  // background
+  --pf-c-nav__list-horizontal-link--disabled--BackgroundColor:          var(--pf-global--disabled-color--200);
+  // after
+  --pf-c-nav__list-tertiary-link--after--BackgroundColor:               transparent;
+  --pf-c-nav__list-tertiary-link--active--after--BackgroundColor:       var(--pf-global--link--Color);
+  --pf-c-nav__list-tertiary-link--disabled--after--BackgroundColor:     transparent;
+
+
+  // Subnav links
+  --pf-c-nav__subnav--MarginTop:                                        var(--pf-global--spacer--sm);
+  --pf-c-nav__subnav--MaxHeight:                                        pf-size-prem(600px);
+  // automate this value based on previously set values
+  --pf-c-nav__subnav-item--PaddingLeft:                                 calc(var(--pf-global--spacer--xl) + var(--pf-global--spacer--md));
+
+  // List expand
+  // Set unique font size for dropdown caret based on design spec
+  --pf-c-nav__list-expand--FontSize:                                    pf-font-prem(20px);
+  --pf-c-nav__list-expand--Transition:                                  .1s ease-in-out;
+
+  // Nav section
+  --pf-c-nav__section--MarginTop:                                       var(--pf-global--spacer--xl);
+
+  // Nav section title
+  --pf-c-nav__section-title--PaddingTop:                                var(--pf-global--spacer--sm);
+  --pf-c-nav__section-title--PaddingRight:                              var(--pf-global--spacer--xl);
+  --pf-c-nav__section-title--PaddingBottom:                             var(--pf-global--spacer--sm);
+  --pf-c-nav__section-title--PaddingLeft:                               var(--pf-global--spacer--xl);
+  // Nav section title fonts
+  --pf-c-nav__section-title--FontSize:                                  var(--pf-global--FontSize--sm);
+  --pf-c-nav__section-title--Color:                                     var(--pf-global--Color--dark-200);
+  --pf-c-nav__section-title--FontWeight:                                var(--pf-global--FontWeight--semi-bold);
+  
+  /* stylelint-enable */
+
+  // Base styling
+  overflow-x: auto;
+
+  // Set width when nav is child of sidebar to prevent collapsing
+  @media screen and (min-width: $pf-global--breakpoint--md) {
+    .pf-l-page__sidebar & {
+      width: var(--pf-c-nav--Width);
+    }
+  }
+
+  // Hide scrollbars
+  &::-webkit-scrollbar {
+    display: none;
+  }
+
+  // List default
+  &__list {
```

Comment:
General comment, we just had a discussion yesterday about only using & for nesting `&.`, `&:` but not for `&-` and `&_` and `&[a-z]` in order to increase readability of the code and to preserve the ability to search and find a class name. It's not in the coding guidelines yet but @michael-coker was looking at adding it to the linter.

Reply:
👍